.cirrus.yml

```diff
@@ -181,10 +184,11 @@ task:
       test_web_engine_firefox_script: |
         cd $ENGINE_PATH/src/flutter/web_sdk/web_engine_tester
         $ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/pub get
-        cd $ENGINE_PATH/src/flutter/lib/web_ui
+        cd $ENGINE_PATH/src/flutter/tools/luci
         $ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/pub get
-        export FELT="$ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/dart dev/felt.dart"
-        $FELT test --browser=firefox
+        export LUCI="$ENGINE_PATH/src/out/host_debug_unopt/dart-sdk/bin/dart bin/luci.dart"
+        $LUCI run //lib/web_ui:unit_tests_linux_firefox
+
```

Comment:
I'll remove this soon. I'm carrying it to LUCI: https://github.com/flutter/flutter/issues/55978